docs/guide/repository.md

```diff
@@ -452,20 +454,8 @@ The cache system is exposed as a "cache" service, and can be reused by any other
 
 #### Configuration
 
-By default, configuration currently uses **FileSystem**, with` %kernel.cache_dir%/stash` to store cache files.
-
-The configuration is placed in `app/config/config.yml` and looks like this:
-
-``` yaml
-# Default config.yml
-stash:
-    caches:
-        default:
-            drivers:
-                - FileSystem
-            inMemory: true
-            registerDoctrineAdapter: false
-```
+By default, configuration currently uses **FileSystem** to store cache files, which is defined in [`default_parameters.yml`](https://github.com/ezsystems/ezplatform/blob/2.0/app/config/default_parameters.yml#L22).
```

Comment:
Is it the permalink to this file? (I think so, but have you verified?)

Reply:
Well, the line number might change if there are changes to  the file, but there's no better way to link to it AFAIK.